libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/SpaceAvatar.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.libraries.designsystem.components.avatar
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.element.android.libraries.designsystem.preview.ElementThemedPreview
+import io.element.android.libraries.designsystem.preview.PreviewGroup
+import io.element.android.libraries.designsystem.utils.CommonDrawables
+
+@Composable
+fun SpaceAvatar(
+    avatarData: AvatarData,
+    avatarType: AvatarType.Space,
+    modifier: Modifier = Modifier,
+    isTombstoned: Boolean = false,
```

Comment:
Can we add the `isTombstoned` property to the `AvatarType.Space` too? This way it's consistent with `AvatarType.Room`.

Reply:
Ah yes true, good catch! https://github.com/element-hq/element-x-android/pull/4921/commits/59aec1290672ecefd87c00879cb0c9e845f40898